charger/alfen.go

```diff
@@ -244,9 +251,22 @@ func (wb *Alfen) Currents() (float64, float64, float64, error) {
 	return res[0], res[1], res[2], nil
 }
 
-// phases1p3p implements the api.PhaseSwitcher interface
-func (c *Alfen) phases1p3p(phases int) error {
-	_, err := c.conn.WriteSingleRegister(alfenRegPhases, uint16(phases))
+func (wb *Alfen) is3p() bool {
```

Comment:
This is a bit complex. Lets just check in the New function? Err must be nil and phases match value/nan/nan?

Reply:
This is exactly what the function currently does. Do you want to remove logging and shorten the code?